packages/zowe-explorer/src/extension.ts

```diff
@@ -612,9 +622,13 @@ function initUSSProvider(context: vscode.ExtensionContext) {
         )
     );
     context.subscriptions.push(
-        vscode.commands.registerCommand("zowe.uss.removeSession", async (node: IZoweUSSTreeNode) =>
-            ussFileProvider.deleteSession(node)
-        )
+        vscode.commands.registerCommand("zowe.uss.removeSession", async (node: IZoweUSSTreeNode, nodeList) => {
+            let selectedNodes = getSelectedNodeList(node, nodeList);
```

Comment:
This can also be `const` (see other note)

Reply:
Please see comment above with same context